src/ui/LiveButton.js

```diff
@@ -10,50 +11,79 @@ import {
   UIText,
 } from '@apollosproject/ui-kit';
 import { AnalyticsConsumer } from '@apollosproject/ui-analytics';
+import { PLAY_VIDEO } from '@apollosproject/ui-media-player';
 import {
-  RockAuthedWebBrowser,
   LiveConsumer,
+  RockAuthedWebBrowser,
 } from '@apollosproject/ui-connected';
 
 const LiveCard = styled(({ theme }) => ({
   backgroundColor: theme.colors.lightSecondary,
 }))(Card);
 
+const renderLiveButton = ({ action }) => (
```

Comment:
if you make action here a positional arg, you probably won't need prop types and it'll clean up the code in both render functions

Reply:
Sorry ... just getting back to this. Not sure what you mean by "positional arg". I'm doing a little Googling but if you wanted to give me an example of what you're talking about that would probably be helpful!

Reply:
I might have figured out what you were talking about.

Reply:
sorry, like this `const renderLiveButton = (action) => (`

doesn't need to be an object, turns it into a react component vs just a function. like the other two render fucntions

Reply:
Yeah, I think I did that right. Check that last commit.